software-modules/programming/codeblocks/codeblocks.sh

```diff
@@ -23,6 +23,10 @@ apt autoremove --yes
 
 ## Prepare final files
 cp ./codeblocks.desktop /usr/share/applications/
+mkdir -p /home/contestant/.config/codeblocks/
+chown contestant /home/contestant/.config/codeblocks/
```

Comment:
Probably better to move this chown after line 29 and do it recursively for all the directory so that any other file in the future is already chowned

Reply:
I get the chown after line 29, but how can I do it recursivelly? :thinking: